### PR TITLE
overlook some plugins

### PR DESCRIPTION
**Pull request by cgzones**

### CODE REVIEW

File: plugins/node.d/df [hunk 1]
```diff
@@ -24,7 +26,7 @@ The following environment variables are used by this plugin:
 . "$MUNIN_LIBDIR/plugins/plugin.sh"
 
 
-df_entries=$(df -P -l 2>/dev/null | sed -e 1d -e '/\/\//d' -e 's/%//' | sort)
+df_entries=$(df -P -l 2>/dev/null | sed -e '1d' -e '/\/\//d' -e 's/%//' | sort) || true
```
Comment (sumpfralle) [inline plugins/node.d/df]: Do you have a specific error in mind, that should be tolerated in this line?
I can only imagine a failed `df` invocation. And I guess, we would not want to ignore that?

File: plugins/node.d/df_inode [hunk 2]
```diff
@@ -30,7 +32,7 @@ The following environment variables are used by this plugin:
 . "$MUNIN_LIBDIR/plugins/plugin.sh"
 
 
-df_entries=$(df -P -l -i -x btrfs 2>/dev/null | sed -e '1d' -e '/\/\//d' -e 's/%//' | sort)
+df_entries=$(df -P -l -i -x btrfs 2>/dev/null | sed -e '1d' -e '/\/\//d' -e 's/%//' | sort) || true
```
Comment (sumpfralle) [inline plugins/node.d/df_inode]: See above.

**Comment by sumpfralle** [comment]
What a nice cleanup!

Please take a look at my two nitpicks below.

Besides this: could you please apply these changes to the `stable-2.0` branch instead? Thank you!

File: plugins/node.d/df_inode (hunk 2 above)
Comment (cgzones) [reply plugins/node.d/df_inode]: we do not want to terminate the script here (with the introduction of `set -e`)

File: plugins/node.d/df (hunk 1 above)
Comment (cgzones) [reply plugins/node.d/df]: without the quotes it does not work in my sandbox (btw. df_inode has the quotes too)

Comment (sumpfralle) [reply plugins/node.d/df]: The quotes are fine (but they should not make a difference). I meant `|| true`.

File: plugins/node.d/df_inode (hunk 2 above)
Comment (sumpfralle) [reply plugins/node.d/df_inode]: Yes, I understand. But I cannot imagine, which part of this line could fail (for a good reason that we want to ignore). What do you think?

**Comment by cgzones** [comment]
You're right; updated

**Comment by sumpfralle** [comment]
Good!
Could rebase (or apply) your changes on `stable-2.0`?

btw.: feel free to just force-push your new branch onto an old instead of opening a new pull request. But this is just a matter of taste ...

**Comment by cgzones** [comment]
opening a new one seems to be easier for me (see #1016)

**Pull request closed without merging**